pod-operation/src/state_machine.rs

```diff
@@ -11,13 +11,13 @@ use tracing::info;
 
 const TICK_INTERVAL: Duration = Duration::from_millis(500);
 
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, enum_map::Enum)]
 pub enum State {
-	Start,
-	Stop,
-	ForceStop,
-	Load,
 	Init,
+	Load,
+	Running,
+	Stopped,
+	Halted,
```

Comment:
Question: While these state names are more descriptive, I feel like they make it a bit more confusing because on the GUI, the buttons to change the state are using the old state names. Were you planning on changing the state names in the frontend too?

Reply:
The control panel buttons represent pod actions, so I don't think there should necessarily be a 1:1 correspondence, especially when the state machine will also make its own transition decisions. It wouldn't be quite right to name the state "Starting" but I guess we could change the button to "Run"? We also don't have an "Init" button because the pod should initialize itself. I do agree that we could rename the "Force Stop" button to "Halt" though.

Reply:
any way we can leave them to match the GUI? i think it would help anyone who visits the code later match whats happening

Reply:
I'll update the buttons to match the base verb, but the point still stands: there will be more states than action buttons. Another example is if a sensor or other component encounters an error, transitioning to a Fault state would be more appropriate than regular Stopped.